common/Helpers/RuntimeHelper.cs

```diff
@@ -34,4 +34,12 @@ public static bool IsCurrentProcessRunningAsAdmin()
         var identity = WindowsIdentity.GetCurrent();
         return identity.Owner?.IsWellKnown(WellKnownSidType.BuiltinAdministratorsSid) ?? false;
     }
+
+    public static void VerifyCurrentProcessRunningAsAdmin()
+    {
+        if (!IsCurrentProcessRunningAsAdmin())
+        {
+            throw new UnauthorizedAccessException("This operation requires elevated privileges.");

```

Comment:
Should this be localizable?

Reply:
No... this is a programming error. I wouldn't expect this to surface to an end user.